controller/array_action/array_mediator_xiv.py

```diff
@@ -211,6 +211,25 @@ def delete_volume(self, volume_id):
 
         logger.info("Finished volume deletion. id : {0}".format(volume_id))
 
+    def delete_snapshot(self, snapshot_id):
+        logger.info("Deleting snapshot with id : {0}".format(snapshot_id))
+        try:
+            snapshot_name = self._get_vol_by_wwn(snapshot_id)
```

Comment:
add log with snapshot name to log

Reply:
there is one in "_get_vol_by_wwn" function. should we have two consecutive and similar logs?

Reply:
didn't see it. Ok ignore